admin/views/tabs/metas/taxonomies.php

```diff
@@ -44,7 +44,11 @@
 			$taxonomies_help->get_button_html() . $taxonomies_help->get_panel_html()
 		);
 
-		$editor = new WPSEO_Replacevar_Editor( $yform, 'title-tax-' . $tax->name, 'metadesc-tax-' . $tax->name );
+		// Determine the post type for the term, this is needed for the recommended replacement variables.
+		$recommended_replace_vars = new WPSEO_Admin_Recommended_Replace_Vars();
+		$post_type = $recommended_replace_vars->determine_for_term( $tax->name );
```

Comment:
Can you align the `=`

Also in this case it isn't a post_type. I think page_type world be better